chef/cookbooks/network/recipes/default.rb

```diff
@@ -16,6 +16,8 @@
 
 return if node[:platform_family] == "windows"
 
+dirty = false
```

Comment:
I will leave the comments here, as github doesn't let me comment on areas that are not in the hunks for some reason:

These links should refer to lines in the code that do `node.set` and I think we are missing `dirty` there.
https://github.com/Itxaka/crowbar-core/blob/2dabd459013cb90ea3571637cb0538ae146d44b4/chef/cookbooks/network/recipes/default.rb#L153